### PR TITLE
OHRM5X-1250: upgrader screen pages

### PR DESCRIPTION
**Pull request by Ajiharan007**
this PR contains
OHRM5X-1250 : upgrader screen added
new rule added
fix styles
api added

### CODE REVIEW

File: installer/Controller/UpgraderProcessController.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+{
+    /**
+     * @inheritDoc
+     */
+    public function preRender(Request $request): void
+    {
+        $component = new Component('upgrade-process-screen');
+        $this->setComponent($component);
+    }
+}
```
Comment (Super-Chama) [inline installer/Controller/UpgraderProcessController.php:35]: ?

File: installer/client/src/pages/UpgradeScreen.vue [hunk 2]
```diff
@@ -17,23 +36,32 @@
       <oxd-form-row>
         <oxd-grid :cols="3" class="orangehrm-full-width-grid">
           <oxd-grid-item class="orangehrm-upgrade-process-item">
-            <oxd-text tag="h6">Applying Database Changes</oxd-text>
-            <oxd-icon name="check-circle-fill"></oxd-icon>
+            <oxd-text :class="getDatabaseClass" tag="h6"
+              >Applying Database Changes</oxd-text
```
Comment (Super-Chama) [inline installer/client/src/pages/UpgradeScreen.vue:40]: manual format

File: installer/client/src/pages/UpgradeScreen.vue [hunk 3]
```diff
@@ -17,23 +36,32 @@
...
+            ></oxd-icon>
+            <oxd-icon v-else name="circle"></oxd-icon>
           </oxd-grid-item>
         </oxd-grid>
         <oxd-grid :cols="3" class="orangehrm-full-width-grid">
           <oxd-grid-item class="orangehrm-upgrade-process-item">
-            <oxd-text tag="h6">Creating Configuration files</oxd-text>
-            <oxd-icon name="circle"></oxd-icon>
+            <oxd-text tag="h6" :class="getConfigClass"
+              >Creating Configuration files</oxd-text
```
Comment (Super-Chama) [inline installer/client/src/pages/UpgradeScreen.vue:52]: format

**Review by Super-Chama — CHANGES_REQUESTED** [review]

File: installer/Controller/UpgraderProcessController.php (hunk 1 above)
Comment (Ajiharan007) [reply installer/Controller/UpgraderProcessController.php:35]: upgrader screen

Comment (Super-Chama) [reply installer/Controller/UpgraderProcessController.php:35]: yes but no new line at end.

**Review by Super-Chama — APPROVED** [review]

**Review by RajithaKumara — APPROVED** [review]

**Pull request merged**